index.html

```diff
@@ -2891,6 +2896,66 @@ <h3 id="applying">
         </ol>
       </section>
     </section>
+      <section>
+        <h3>
+          <code title="">iarc_rating_id</code> member
```

Comment:
Looking at the cert you sent us, I wonder if we should actually rename this "iarc_certificate_id"? It's not a rating id, it's actually the certificate UUID that is unique for this particular application, no? Or did I misunderstand.